aya/CHANGELOG.md

```diff
@@ -182,1097 +182,708 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Chore
 
- - <csr-id-b3e7ef741c5b8d09fc7dc8302576f8174be75ff4/> Use the cargo workspace package table
```

Comment:
Why?

Reply:
Oops, zed just reformatted this without asking me, reverted